### PR TITLE
Add conf dates to footer+header

### PR DESCRIPTION
**Pull request by apiraino**

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -1,11 +1,9 @@
 <header class="navigation" role="banner">
   <section class="itemgroup">
-    <a href="/" class="logo" title="RustFest">
+    <a href="/" class="logo" title="RustFest" alt="RustFest">
       {% include icons/logo-header.svg %}
-      <span class="site-title">
-        RustFest
-      </span>
     </a>
+    <span style="color:white;">{{ site.location.city }}, {{ site.location.country }} - {{ site.dates.short_form }}</span>
```
Comment (apiraino) [inline _includes/header.html]: I don't like how the `ul` at line 8 pushes this `span` on two lines. Can any CSS wizard help out here?

File: _config.yml [hunk 2]
```diff
@@ -26,6 +26,7 @@ dates:
   workshops: "Sunday, November 25th"
   impl_days: "TBA"
   impl_days_no_br: "TBA"
+  short_form: "Nov. 24th-25th 2018"
```
Comment (dns2utf8) [inline _config.yml]: IMHO it is a comma after Nov

File: _includes/header.html (hunk 1 above)
Comment (dns2utf8) [reply _includes/header.html]: without testing, have you checked if the `section.itemgroup` has a `max-height`?
Another one might be `clear: both;` if there was a `float` somewhere

**Review by dns2utf8 — DISMISSED** [review]

File: _includes/header.html (hunk 1 above)
Comment (dns2utf8) [reply _includes/header.html]: Also `style=...` should go into `_sass/components/_navigation.scss` around line 11.

File: _config.yml (hunk 2 above)
Comment (apiraino) [reply _config.yml]: fixed!

**Comment by apiraino** [comment]
@dns2utf8 I've dug the thing to the bottom and finally understood a little bit more how SASS compilation works ;-)

I've applied some styling that shrinks a little bit the link menu in the header and the social icons, so now we have enough room for the additional text about conf.dates

**Comment by tglman** [comment]
Hi,

Yes this now looks a bit better, the top menu now looks a bit too crowded, probably just a `hover` on the header menu may improve a bit the situation, the color of the `hover` on the footer is actually making it  a bit worst, I'll see if i can improve it :)

**Comment by badboy** [comment]
Hm, yeah, it's definitely too crowded in the header now and the links are not visible as such and too close together as well.
I don't have a good idea on how to fix that though.

**Comment by apiraino** [comment]
@badboy I agree with you. So, if we don't want to squeeze the header elements, I think our only option is to add another line. Here's 4 proposals (tbh none of them is really exciting - but yeah, I'm out of options here):

prop 0
Proposal no. 0

prop 1
Proposal no. 1

prop 2
Proposal no. 2

prop 3
Proposal no. 3

Proposal no. 4: leave this issue to someone more knowledgeable than me ;-)

**Comment by tglman** [comment]
I did add the hover, that make it looks better anyway I like more your proposal 1 and 3

**Comment by badboy** [comment]
yeah, 1 could work. Did you check how it looks on mobile? (Chrome & Firefox have ways to simulate the size of an iPhone or Galaxy Ssomething in the dev tools)

**Comment by apiraino** [comment]
ok loaded the header proposal and tested on mobile, looks ok.

I tried to improve also the footer a bit, but whatever I touched broked the mobile view (thanks for reminding me to check mobile view in the browser!)

**Comment by flaki** [comment]
> @badboy I agree with you. So, if we don't to squeeze the header elements, I think our only option is to add another line. Here's 4 proposals (tbh none of them is really exciting - but yeah, I'm out of options here):
> 
> prop 0
> Proposal no. 0
> 
> prop 1
> Proposal no. 1
> 
> prop 2
> Proposal no. 2
> 
> prop 3
> Proposal no. 3
> 
> Proposal no. 4: leave this issue to someone more knowledgeable than me ;-)

My proposal:

> 2018-11-24 & 25, Rome :it:

or alternatively

> 2018/11/24–25, Rome :it:

Text under the social icons.


Also just don't worry _too much_ about this, just get it done ;) The website really needs a proper visual & UX overhaul at some point anyway.

**Comment by apiraino** [comment]
ok done and looks pretty cool - also, yes please move on to other things :)

untitled

**Review by badboy — APPROVED** [review]

**Comment by badboy** [comment]
:shipit:

**Comment by tglman** [comment]
That's looks great to me as well.

shall we merge ?

**Pull request merged**